config/slashCommandPermissions.js

```diff
@@ -4,6 +4,9 @@ const {ID_CODE_COUNSELOR, ID_MODERATOR, ID_ADMIN, ID_SUPER_ADMIN} = process.env;
 
 const commandRoles = new Map([
   ['ping', [ID_CODE_COUNSELOR, ID_MODERATOR, ID_ADMIN, ID_SUPER_ADMIN]],
+  ['notes', [ID_CODE_COUNSELOR, ID_MODERATOR, ID_ADMIN, ID_SUPER_ADMIN]],
+  ['addnote', [ID_CODE_COUNSELOR, ID_MODERATOR, ID_ADMIN, ID_SUPER_ADMIN]],
+  ['removenote', [ID_MODERATOR, ID_ADMIN, ID_SUPER_ADMIN]],
```

Comment:
Now that I see this with multiple commands, I'm not a fan of having to list out all the perms.
What would you think of a small refactor where we save these perm sub-sets to variables instead?
E.g., `[ID_CODE_COUNSELOR, ID_MODERATOR, ID_ADMIN, ID_SUPER_ADMIN]` could be something like `allStaffRoles`, and we could also have something like `moderatorRoles`.  I think this would clean up reading through the command roles map quite a bit.

Reply:
Good idea. I refactored the permissions to use this approach as part of this PR and it looks much cleaner.